### PR TITLE
Generate psalm-baseline.xml PR instead of requiring this from the PR …

### PR DESCRIPTION
**Pull request by MorrisJobke**
…author itself

If all works out this will generate a daily PR that contains the changes in psalm-baseline that we then just can merge.

### CODE REVIEW

File: .github/workflows/update-psalm-baseline.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: Static code analysis
```
Comment (go2sh) [inline .github/workflows/update-psalm-baseline.yml:1]: Copy & Pasta? ;-)

Comment (MorrisJobke) [reply .github/workflows/update-psalm-baseline.yml:1]: 🙈

**Comment by MorrisJobke** [comment]
So I tested this in a separate repo: https://github.com/MorrisJobke/server/runs/1006952567?check_suite_focus=true

It only works when after the merge another scheduled action is created via the Github Action web UI. See also the answers in https://stackoverflow.com/questions/59560214/github-action-works-on-push-but-not-scheduled 🙈

**Comment by MorrisJobke** [comment]
https://github.com/MorrisJobke/server/pull/2 has worked :)

**Comment by MorrisJobke** [comment]
I created an account with no permissions here and only a personal fork of the server to send in the PRs.

**Review by ChristophWurst — APPROVED** [review]
:elephant:

**Pull request merged**